devicetypes/erocm123/inovelli-2-channel-smart-plug.src/inovelli-2-channel-smart-plug.groovy

```diff
@@ -83,7 +83,7 @@ def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, ep = null)
 			def allOff = true
 			childDevices.each {
 				n ->
-					if (n.currentValue("switch")? != "off") allOff = false
+					if (n.currentState("switch") != null && n.currentState("switch").value != "off") allOff = false
```

Comment:
Wouldn't `n.currentState("switch")?.value != "off"` also work?

Reply:
Probably.

Reply:
I also seem to recall a conversation with @tpmanley on another PR I had created about the original statement should work even if current value returns null.